modules/caddyhttp/reverseproxy/reverseproxy.go

```diff
@@ -569,6 +569,23 @@ func (h *Handler) proxyLoopIteration(r *http.Request, origReq *http.Request, w h
 	return false, proxyErr
 }
 
+const canonicalWsHeaderPrefix = "Sec-Websocket"
+
+// websocket header replacer, normally server doesn't care about this difference,
+// but some do, and RFC use the case too
+// gorilla/websocket, gobwas/ws all use this case too.
+// see https://caddy.community/t/websockets-fail-venus-os-cerbo-gx-victron/25685
+var wsHeaderReplacer = strings.NewReplacer(canonicalWsHeaderPrefix, "Sec-WebSocket")
+
+func normalizeWsHeader(header http.Header) {
+	for k, v := range header {
```

Comment:
There's only exactly 5 headers we need to care about. I think if we directly check for those headers, it would perform better, because it would avoid having to do string comparisons on every header field.